Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.yml

```diff
@@ -19,6 +19,14 @@ args:
   name: forced_encoding
   description: Use only the force encoding when parsing the message, e.g 'iso-2022-jp'.
     Relevant to msg files only.
+- auto: PREDEFINED
+  defaultValue: "All files"
+  description: Decides how many email files to parse, for multiple nested email files (for instance - an EML inside an EML).
```

Comment:
```suggestion
  description: In case of nested email files (for instance, an EML file inside an EML file), determines which of the email files to return as an output.
```